app/controllers/ops_controller/settings.rb

```diff
@@ -153,6 +153,7 @@ def region_edit
       @edit[:region].description = @edit[:new][:description]
       begin
         @edit[:region].save!
+        MiqRegion.my_region_clear_cache
```

Comment:
Yeah, @carbonin that might be an alternative.  

```ruby
MiqRegion.my_region_clear_cache if MiqRegion.my_region_number == self.region
```

in an after_save.  What do you think @chrisarcand @bdunne?